src/Extensions/Logging/Internal/OmexLogsEventSource.cs

```diff
@@ -0,0 +1,278 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Diagnostics.Tracing;
+using System.Globalization;
+using System.Text;
+using Microsoft.Extensions.Logging;
+using Microsoft.Omex.Extensions.Abstractions;
+using Microsoft.Omex.Extensions.Logging.Replayable;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	[EventSource(Name = "Microsoft-OMEX-Logs")]
+	internal sealed class OmexLogsEventSource : EventSource
+	{
+		static OmexLogsEventSource()
+		{
+			Process process = Process.GetCurrentProcess();
+			s_processName = string.Format("{0} (0x{1:X4})", process.ProcessName, process.Id);
+		}
+
+
+		public OmexLogsEventSource(IMachineInformation machineInformation, IServiceContext context)
+		{
+			m_applicationName = machineInformation.MachineRole;
+			m_serviceName = machineInformation.ServiceName;
+			m_buildVersion = machineInformation.BuildVersion;
+			m_machineId = machineInformation.MachineId;
+
+			m_partitionId = context.PartitionId;
```

Comment:
Is this blank line deliberate?

Reply:
It was, but there is no point in it when I'm thinking about it now, removed, thanks